source/MRMesh/MRObjectLoad.cpp

```diff
@@ -668,7 +717,7 @@ Expected<std::shared_ptr<Object>, std::string> loadSceneFromAnySupportedFormat(
     for ( auto& c : ext )
         c = ( char )tolower( c );
 
-    auto res = unexpected( std::string( "unsupported file extension" ) );
+    Expected<std::shared_ptr<Object>, std::string> res = unexpected( std::string( "unsupported file extension" ) );
```

Comment:
can be just `Expected<std::shared_ptr<Object>>` (std::string is default error type)